lib/salus/scanners/report_base.rb

```diff
@@ -0,0 +1,8 @@
+module Salus::Scanners
+  # Super class for all scanner objects.
+  class ReportBase < Base
```

Comment:
Is there something that ensures this file is loaded before all classes that inherit from it?
Maybe by chance, it would work ok, but files may be loaded in random orders.
Perhaps it would be best to explicitly `require` this file first.

Reply:
Good point, Done and added explicit "requires"